documentation/src/main/asciidoc/userguide/chapters/domain/entity.adoc

```diff
@@ -117,6 +117,13 @@ The main piece in mapping the entity is the `javax.persistence.Entity` annotatio
 The `@Entity` annotation defines just one attribute `name` which is used to give a specific entity name for use in JPQL queries.
 By default, the entity name represents the unqualified name of the entity class itself.
 
+[IMPORTANT]
+====
+Because the entity name is given by the unqualified name of the class, Hibernate does not allow registering multiple entities with the same name even if the entity classes reside in different packages.
+
+Without imposing this restriction, Hibernate would not know which entity class is referenced in a JPQL query if the unqualifid entity name is associated with more then one entity classes.
```

Comment:
s/unqualifid/unqualified/

Maybe we should say a word about specifying a different name in `@Entity`?

Reply:
I'll check that.